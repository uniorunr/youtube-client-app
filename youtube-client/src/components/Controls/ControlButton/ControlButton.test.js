import controlButton from './ControlButton';

describe('controlButton', () => {
  it('should be instance of Function', () => {
    expect(controlButton).toBeInstanceOf(Function);
  });

  it('should be rendered correctly', () => {
    const button = controlButton('test-class', 'test-inner-html');
    document.body.appendChild(button);
    expect(document.body.innerHTML).toMatchSnapshot();
  });
});
