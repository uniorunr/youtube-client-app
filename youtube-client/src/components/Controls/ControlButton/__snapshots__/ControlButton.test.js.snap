// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`controlButton should be rendered correctly 1`] = `"<button class=\\"test-class\\" type=\\"button\\">test-inner-html</button>"`;
